src/modules/account/hooks/useDeprecatedAccount.js

```diff
@@ -0,0 +1,178 @@
+/* istanbul ignore file */
+import { useEffect, useState } from 'react';
+import { useDelegates, useSentVotes, useUnlocks } from '@dpos/validator/hooks/queries';
+import { useAuth } from '@auth/hooks/queries';
+import { useLegacy } from '@legacy/hooks/queries';
+import { useDispatch } from 'react-redux';
+import authActionTypes from '@auth/store/actionTypes';
+import { useCurrentAccount } from './useCurrentAccount';
+
+// eslint-disable-next-line import/prefer-default-export,max-statements
+export const useDeprecatedAccount = () => {
+  const [currentAccount] = useCurrentAccount();
+  const dispatch = useDispatch();
+  const { pubkey, address } = currentAccount.metadata;
+  const [account, setAccount] = useState({
+    summary: {
+      address,
+      publicKey: '',
+      legacyAddress: '',
+      // balance: @todo account has multiple balance now
```

Comment:
why is this todo? we can store the balances in the state and later we can adjust the implementation?

Reply:
maybe @reyraa can make better judgement here.